### ISSUE TITLE
[Broken Website] speaky.com

### ISSUE THREAD

**Issue by timea-techgirl** [darkreader/darkreader#8387]
# Broken Website Report

## Website Address
www.speaky.com

## Steps To Reproduce
- Go to speaky.com and **login**
- Refresh or reload the page
- You will see that the website shows light mode. I have to toggle the dark mode off and on to fix it on each reload. Rarely it reloads fine.
- On a fresh browser profile with no other extensions, the same issue starts occurring. The first time, dark mode may work fine but the issue starts again.

## Problematic Observation
I have experienced the problem only on speaky.com. I have had this problem since many versions but long ago, this problem didn't exist. I tried changing settings and many combinations, nothing seem to fix the issue for this site.

## System Information
- OS: Windows
- Browser: Chrome 97
- Dark Reader Version: 4.9.45

**Comment by Gusted** [comment]
Hi @timea-techgirl!

Have you tried to check with other extensions, it could be that one of them is conflicting with Dark Reader.
You could also try to create a new temporary profile to see if this reproduces(it doesn't for me).

Regards,
Gusted

**Comment by timea-techgirl** [comment]
@Gusted Yes, I've tested many times with a new chrome profile and also incognito tab. As I mentioned, the issue starts occurring after sometime.

**Comment by Gusted** [comment]
That's quite weird - have been trying to get it reproduced, but it's looking good on my end.
Does the console(ctrl + shift + i -> Console) show any errors?

**Comment by timea-techgirl** [comment]
I don't think any errors are related to dark reader. When this happens, the scroll bar is dark, rest of the page is white.
![image](https://user-images.githubusercontent.com/7058205/157131600-4e710f54-aa9c-4c84-a3fd-81a9db780130.png)

**Comment by Gusted** [comment]
Hmm I don't really have a good clue or a lead what could be going on - as I'm still not really able to reproduce this bug. What the sites does isn't that special and from my looks it should be just able to handle this site as intended.

**Comment by timea-techgirl** [comment]
I have no idea how it is working for you.
I have a simple setup, I just tried creating a new profile again and the issue exists.
Just to be sure, I did it on a fresh virtual machine and was able to reproduce the issue. Opened speaky, opened the messages tab in new window, reloaded pages. Reloads to white theme 95 percent of the times. There are no other extensions installed.

![image](https://user-images.githubusercontent.com/7058205/157312973-067624ab-ce54-4ff8-a612-4a86435d2185.png)

![image](https://user-images.githubusercontent.com/7058205/157312896-547b59c7-7cfe-48f1-9298-8e41126142df.png)

**Comment by Gusted** [comment]
For what's worth the site also seems to be doing some things with useragents and user's OS(I'm using Linux) so that maybe could be something but that would be quite weird. Might be something on my site that "correctly fixes it" - but then I lost the clue of what dark reader is doing incorrectly here.
